funded_coldkeys.txt

```diff
@@ -0,0 +1,186 @@
+5EjcsEVGp38zui7y9ED38qHqgauBycH4kB9m5xfNstnYG2py
```

Comment:
not sure if we should track this file since otherwise we should push every time we update it and this is a tools repo rather than ops from my POV. Maybe we should consult with the team here 🤔

Reply:
yes, let's not commit data here - we can send the data over in different ways and just share the scripts here